interactions/client/context.py

```diff
@@ -75,48 +84,81 @@ def deferred_ephemeral(self) -> bool:
             and self.message.flags & MessageFlags.LOADING
         )
 
-    def __attrs_post_init__(self) -> None:
-        if self.member and self.guild_id:
-            self.member._extras["guild_id"] = self.guild_id
+    @property
+    def created_at(self) -> datetime:
+        """
+        .. versionadded:: 4.4.0
 
-        self.author = self.member
+        Returns when the interaction was created.
+        """
+        return self.id.timestamp
 
-        if self.user is None:
-            self.user = self.member.user if self.member else None
+    @property
+    def author(self) -> Member:
+        """
+        Returns the author/member that invoked the interaction.
+        """
+        return self.member
```

Comment:
What do you think about make it return `self.user` if member is none?

Reply:
That's breaking so no

Reply:
okay then